src/providers/wcs/qgswcscapabilities.h

```diff
@@ -114,6 +123,9 @@ class QgsWcsCapabilities : public QObject
      */
     void coverageParents( QMap<int, int> &parents, QMap<int, QStringList> &parentNames ) const;
 
+    // Parse metadata element from the document
```

Comment:
Better make this a docstring

```suggestion
    //! Parse metadata element from the document
```